download-browser.js

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-async function downloadBrowser(browser) {
-  const browserType = require('.')[browser];
+async function downloadBrowser(browserType) {
```

Comment:
Let's maybe comment that we are passing a browserType instance from the right package to respect the downloadPath?

Reply:
I want to follow-up and actually have this method as part of the API, behind the option:

```js
await playwright.chromium.downloadBrowserIfNeeded({progressbar: true});
```

This way there will be no confusion